src/Carriers/EntitiesCollection.php

```diff
@@ -52,7 +52,7 @@ protected function &variables(array $requiredItems = null) {
             foreach ($requiredItems as $key => $requiredItem) if (isset($variables[$requiredItem])) unset($requiredItems[$key]);
         }
 
-        if (!$this->entitiesAreLoaded AND ((\count($requiredItems) !== 0) OR $requiredItems === null)) {
+        if (!$this->entitiesAreLoaded AND ($requiredItems === null OR \count($requiredItems) !== 0)) {
```

Comment:
maybe better `if (!$this->entitiesAreLoaded AND ($requiredItems === null OR !$requiredItems))`

Reply:
Ok, condition was changed only because parameter to count() function has to be array, not null in this case.

Reply:
The condition changed.